scripts/zol2zfs-patch.sed

```diff
@@ -0,0 +1,17 @@
+#!/bin/sed -f
```

Comment:
Would be nice to determine the sed path automatically. 
For example, I had to change this to _/usr/bin/sed_ when running on my mac.